experiment/generate_report.py

```diff
@@ -52,6 +52,13 @@ def get_arg_parser():
         action='store_true',
         default=False,
         help='If set, plots are created faster, but contain less details.')
+    parser.add_argument(
+        '-p',
+        '--track_progress',
```

Comment:
progress seems confusing to me, since we can do the comparison at end of experiment too.

--compare_experiments ?
--diff_experiments or something feels a little more clear. @lszekeres - thoughts ?

Reply:
I switched to the more explicit "label-by-experiment"

Reply:
label-by-experiment sgtm. perhaps label-fuzzers-by-experiment might be clearer though.